visual-tests/imports/helpers.js

```diff
@@ -0,0 +1,67 @@
+export const helpers = {
+  defaultHOTWrapper: 'js',
+
+  screenshotsDirectory: './tests/screenshots',
+  screenshotsExtension: 'png',
+
+  selectors: {
+    mainTable: 'div.handsontable.htRowHeaders.htColumnHeaders',
+    mainTableBody: '> .ht_master.handsontable table tbody',
+    mainTableHead: '> .ht_clone_top.handsontable table thead',
+    mainTableFirstColumn: '> .ht_clone_inline_start.ht_clone_left.handsontable table tbody',
+    dropdownMenu: '.htMenu.htDropdownMenu.handsontable',
+  },
+
+  cssFiles: {
+    cookieInfo: 'remove-cookie-info.css',
+    changeCellsBackground: 'change-cells-background.css',
+    changeCellTextColor: 'change-cell-text-color.css',
+    dynamicDataFreeze: 'dynamic-data-freeze.css'
+  },
+
+  expectedPageTitle: /Handsontable for .* example/,
+
+  cssPath(file) {
+    return `./tests-css/${file}`;
+  },
+
+  init(workerInfo, process) {
+    this.hotWrapper = process.env.HOT_WRAPPER || this.defaultHOTWrapper;
+    this.testURL = 'http://localhost:8080/';
+    this.workerInfo = workerInfo;
+    this.isMac = workerInfo.project.name === 'webkit';
+    this.modifier = this.isMac ? 'Meta' : 'Control';
+    this.screenshotsCount = 0;
+    this.screenshotDirName = this.workerInfo.titlePath[0].split('.spec.js')[0];
+    this.browser = this.workerInfo.project.name;
+
+    if (this.browser === 'webkit') {
+      this.browser = 'safari';
+    }
+
+    if (this.browser === 'chromium') {
+      this.browser = 'chrome';
+    }
```

Comment:
```suggestion
```
IMO I'd remove that lines, as I think it's better to name screenshots by the origin engine name, not by the browser. `chromium` engine, can differ from `chrome` and `webkit` does not necessarily mean Safari. This can be confusing while debugging or trying to reproduce the potential test fails.

Reply:
For me this naming is not necessary, but there was two reasons of it:
- my idea was it would be just more obvious for less technical people who not always may know name `webkit`
- one of tested by us services comparing screenshots was detecting browser by filenames and putting them to given categories. I'm not sure after this time, but as far as I remember Webkit was not displayed correctly - it was categorized as Edge.

Since 2 reason is outdated (cause of few obstacles we decided to use another service) I think I can recover previous names.